Editor/Resources/EditorWindow/GameLiftPlugin.cs

```diff
@@ -0,0 +1,167 @@
+            GetWindow().OpenTab(Pages.Credentials);
+        }
+
+        [MenuItem("Amazon GameLift/Host with Anywhere", priority = 101)]
+        public static void OpenAnywhereTab()
+        {
+            GetWindow().OpenTab(Pages.Anywhere);
+        }
+
+        [MenuItem("Amazon GameLift/Host with Managed EC2", priority = 102)]
+        public static void OpenEC2Tab()
+        {
+            GetWindow().OpenTab(Pages.ManagedEC2);
+        }
+
+        [MenuItem("Amazon GameLift/Import Sample Game", priority = 103)]
+        public static void ImportSampleGame()
+        {
+            string filePackagePath = $"Packages/{Paths.PackageName}/{Paths.SampleGameInPackage}";
+            AssetDatabase.ImportPackage(filePackagePath, interactive: true);
+        }
+
+        [MenuItem("Amazon GameLift/Help/Documentation", priority = 200)]
+        public static void OpenDocumentation()
+        {
+            Application.OpenURL(Urls.AwsHelpGameLiftUnity);
+        }
+
+        [MenuItem("Amazon GameLift/Help/AWS GameTech Forum", priority = 201)]
+        public static void OpenGameTechForums()
+        {
+            Application.OpenURL(Urls.AwsGameTechForums);
+        }
+
+        [MenuItem("Amazon GameLift/Help/Report Issues", priority = 202)]
+        public static void OpenReportIssues()
+        {
+            Application.OpenURL(Urls.GitHubAwsLabs);
+        }
+
+        private void CreateGUI()
+        {
+            _root = rootVisualElement;
+            _visualTreeAsset = UnityEngine.Resources.Load<VisualTreeAsset>("EditorWindow/GameLiftPlugin");
+            if (_visualTreeAsset == null)
+            {
+                return;
+            }
+
+            VisualElement uxml = _visualTreeAsset.Instantiate();
+            _root.Add(uxml);
+            
+            ApplyText();
+
+            _tabButtons = _root.Query<Button>(className: TabButtonClassName).ToList();
+            _tabContent = _root.Query(className: TabContentClassName).ToList();
+
+            _tabButtons.ForEach(button => button.RegisterCallback<ClickEvent>(_ => { OpenTab(button.name); }));
+        }
+
+        private void ApplyText()
+        {
+            var l = new ElementLocalizer(_root);
+            l.SetElementText("Landing", Strings.TabLanding);
+            l.SetElementText("Credentials", Strings.TabCredentials);
+            l.SetElementText("Anywhere", Strings.TabAnywhere);
+            l.SetElementText("ManagedEC2", Strings.TabManagedEC2);
+            l.SetElementText("Help", Strings.TabHelp);
+        }
+
+        private void SetElementText(string elementName, string text)
+        {
+            var button = _root.Q<TextElement>(elementName);
+            if (button != default)
+            {
+                button.text = _textProvider.Get(text);
+            }
+        }
+
+        private void OpenTab(string tabName)
+        {
+            _tabContent.ForEach(page =>
+            {
+                if (!page.name.StartsWith(tabName))
```

Comment:
Because they represent different things. The buttons are named exactly for the page, and the content is suffixed with `Tab`

Reply:
Ah I see this is because we're using `${PAGE_NAME}Tab` for our content objects. For the sake of consistency and clarify, consider using `${PAGE_NAME}Button` for the button names

Reply:
If we swap over to this, then we will need a different solution for registering the button callbacks, as it currently uses the button names.

```cs
_tabButtons.ForEach(button => button.RegisterCallback<ClickEvent>(_ => { OpenTab(button.name); }));
```

Something like 

```cs
_tabButtons.ForEach(button => button.RegisterCallback<ClickEvent>(_ => { OpenTab(button.name.Replace("Button", ""));}));
```

Reply:
As a side, can we swap these conditions to positive checks instead of negative checks? It removes the need for a double-negative around the else statement

Reply:
Both of these issues addressed